Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Payment.hs

```diff
@@ -203,6 +204,8 @@ juspayWebhookHandler merchantShortId mbOpCity mbServiceName authData value = do
   case osr of
     Payment.OrderStatusResp {..} -> do
       order <- QOrder.findByShortId (ShortId orderShortId) >>= fromMaybeM (PaymentOrderNotFound orderShortId)
+      whenJust payerVpa $ \vpa -> QDI.updatePayoutVpa (Just vpa) (cast order.personId)
```

Comment:
This handler will be there for other payments too, do we need to update that vpa too in payout?

Reply:
modified